File: java101/Alapok/2017.10.08 vezérlés-while,for/darts/src/darts/Darts.java
package darts;

import java.util.Scanner;

public class Darts {

    public static void main(String[] args) {
        Scanner bill = new Scanner(System.in);
        System.out.println("Darts játék");
        int pont = 0;
        int db = 0;
        int legnagyobb = 0;
        int legkissebb = 60;
        int dobas;
        while (pont != 301) {            
            System.out.println("Pont: " + pont);
            System.out.print("Dobás: ");
            dobas = bill.nextInt();
            db++;
            if (pont + dobas <= 301) {
                pont += dobas;
                if (dobas > legnagyobb) {
                    legnagyobb = dobas;
                }
                if (dobas < legkissebb){
                    legkissebb = dobas;
                }
            }
        }
        System.out.println("Nyertél!");
        System.out.println("Megszámlálás tétele: A dobások száma: " + db);
        System.out.println("Keresés tétele: A legnagyobb dobás: " + db);
        System.out.println("Keresés tétele: A legkissebb dobás: " + db);
    }
}

File: python101/py 19 Ciklusok 4/Források/darts.py
# Ki: Darts játék
# pont = 0
# Amíg pont != 0
#     Ki: pont
#     Be: dobas
#     Ha pont+dobas <= 301
#         pont += dobas
# Ki: Nyertél!

print('Darts játék')
pont = 0
while pont != 301:
    print('Pont: ',pont)
    dobas = int(input('Dobás: '))
    if pont+dobas <= 301:
        pont += dobas
print('Nyertél!')